3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```diff
@@ -17,27 +17,41 @@
 ------------------------------------------------------------------------------*/
 function requestData(url) {
   // TODO return a promise using `fetch()`
+  return fetch(url)
+  .then((respond)=>{
+    return respond.json()
+  })
+  .catch((error)=>{
+    console.log(error)
+  })
```

Comment:
We won't show the nice error message in case of this error. I suggest replace the `console.log` with a function that properly displays the error message